packages/nimble-components/src/rich-text-viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -1,6 +1,25 @@
+import { html } from '@microsoft/fast-element';
```

Comment:
In some other tests we've used [a shared list of wacky strings](https://github.com/ni/nimble/blob/12463598a33340eb311356d29b49d1195d228d86/packages/nimble-components/src/utilities/tests/wacky-strings.ts#L4) to throw lots of crazy inputs at a component and ensure it behaves well. I think we should use the same list for this component.

Here's an example of it being used in a test:https://github.com/ni/nimble/blob/12463598a33340eb311356d29b49d1195d228d86/packages/nimble-components/src/table-column/text/tests/table-column-text.spec.ts#L198

Reply:
I have added this test case as well in this PR.